roles/lib_openshift/src/class/oc_adm_csr.py

```diff
@@ -38,11 +38,13 @@ def create_nodes(self, nodes):
         results = self._get(resource='nodes')['results'][0]['items']
 
         for node in nodes:
-            nodes_list.append(dict(name=node, csrs={}, accepted=False, denied=False))
+            nodes_list.append(dict(name=node, csrs={}, server_accepted=False, client_accepted=False, denied=False))
 
+            # Nodes that we were able to retrieve via "oc get nodes" are approved so mark them approved.
```

Comment:
That's the case for client certs but not server certs. That's part of what's made this problem so hard to track down is that the nodes show up and they're marked as Ready because their client cert is able to post status back to the API, but as soon as the API tries to connect to the kubelet things fail because there's no server cert.

Reply:
We can come back to this later.